src/include/nodes/execnodes.h

```diff
@@ -682,6 +682,9 @@ typedef struct EState
 	 */
 	bool		gp_bypass_unique_check;
 
+	/* List of cross-slice SharedScan consumers in the current slice */
+	List	   *sharedScanConsumers;
```

Comment:
Why didn't this break the ABI tests?

Reply:
Only the size of the struct has changed, it is not used inline in any other structs, so all offsets are unchanged. If you check ABI report, it mentions this as low severity problem.

Reply:
As far as I'm aware, it was issued only for structs that were inlined in other structs and so field offsets were changed. And while that's true, I'm not sure we should prohibit adding fields to structs altogether, a lot of patches would be impossible to implement without adding more data.

Reply:
I just note, that it was cherry-picked from `adb-6.x-dev` to `adb-7.2.0`. The same changes was merged into `adb-6.x-dev`. There was not any problem there.